JavaGenerator/src/org/specs/generators/java/classtypes/ClassType.java

```diff
@@ -246,6 +246,9 @@ public void addImport(JavaType... imports) {
      * @return true if the import can be added, false if not
      */
     public boolean addImport(JavaType newImport) {
+        if (newImport == null) {
+            return false;
+        }
```

Comment:
Are there cases where this is being called with a null value? (vs using a SpecsCheck.checkNotNull()